test/bdd/fixtures/docker-compose.yml

```diff
@@ -62,6 +64,7 @@ services:
     image: ${KMS_REST_IMAGE}:latest
     environment:
       - KMS_HOST_URL=0.0.0.0:8074
+      - KMS_HOST_METRICS_URL=0.0.0.0:48832
```

Comment:
Can't we just use port number here ? it looks like host is always 0.0.0.0

Reply:
I think should be similar to KMS_HOST_URL

Reply:
we do same thing in orb

Reply:
@rolsonquadras